src/App.test.js

```diff
@@ -54,10 +57,12 @@ describe('Signing In', () => {
 
 
   }, 16000);
-  test('login form works correctly', async () => {
-    // take a png screenshot and saves it inside root directory
-    await page.screenshot({ path: "loginScreen.png" })
-    // uses faker to load data from object uptop
+  test('login form leads to the dashboard', async () => {
+
+    // you will need to find the exact path for the screen shots to save in the file. 
+    // i've tried ({path : "../debug_screenshot"}) but you will get a Unsupported screenshot mime type: null error
```

Comment:
`path: "debug_screenshot/login.png"` worked for me (you may have forgotten the extension).

Reply:
you right